tests/test_tomography_gate_set_tomography.py

```diff
@@ -399,3 +254,15 @@ def test_GST(backend, target_gates, Pauli_Liouville):
                 Pauli_Liouville=Pauli_Liouville,
                 backend=backend,
             )
+
+
+def test_GST_invertible_matrix():
+    T = np.array([[1, 1, 1, 1], [0, 0, 1, 0], [0, 0, 0, 1], [1, -1, 0, 0]])
+    matrices = GST(gate_set=[], Pauli_Liouville=True, T=T)
+    assert True
```

Comment:
@AlejandroSopena do you have any idea how to test a different gauge for the matrix `T`? Worst case we could just exclude those lines from coverage.